### PR TITLE
Using environment variable to bump airflow version.

### PR DESCRIPTION
**Pull request by irvifa**

### CODE REVIEW

**Comment by irvifa** [comment]
This is actually more on how we want to define the airflow version, meanwhile https://github.com/GoogleCloudPlatform/cloud-builders-community/pull/425 is to pin the werkzeug version.

File: airflow/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 FROM python:3.7
 
-COPY requirements.txt .
-RUN pip install --no-cache-dir -r requirements.txt
+ARG AIRFLOW_VERSION=UNSPECIFIED
```
Comment (henryken) [inline airflow/Dockerfile:3]: Why do we set it to "UNSPECIFIED"?

File: airflow/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,8 @@
 FROM python:3.7
 
-COPY requirements.txt .
-RUN pip install --no-cache-dir -r requirements.txt
```
Comment (henryken) [inline airflow/Dockerfile:4]: If the requirements.txt is deleted, the #425 would not work anymore

**Comment by henryken** [comment]
The change from #425 and this would introduce conflict that needs to be resolved.

**Comment by irvifa** [comment]
@henryken should I pull from master, rebase, and squashed it into one?

**Comment by henryken** [comment]
@irvifa, we can just wait for any of the pull request to be merged. Then we can fix the other PR accordingly.
Our PRs have been open for quite some time.

**Comment by henryken** [comment]
@irvifa, PR #425 has been merged. Would you be able to incorporate that change to this PR?

**Comment by irvifa** [comment]
@henryken I've rebase it from current upstream master

**Comment by henryken** [comment]
Thank you @irvifa! LGTM.

**Comment by irvifa** [comment]
Hi @henryken will this be merged into master then?

**Comment by henryken** [comment]
It should be. Let's wait for the team to review and merge it.

**Pull request merged**